system-tests/cli/cli.test.js

```diff
@@ -0,0 +1,35 @@
+"use strict";
```

Comment:
Also to start process we can use `execa`

Reply:
OK, I'll use `execa` if necessary.